src/test/data/JsonSerializableAddressBookTest/invalidRecipeAddressBook.json

```diff
@@ -0,0 +1,45 @@
+{
+  "clients": [ {
+    "name": "Hans Muster",
+    "phone": "9482424",
+    "email": "invalid@email!3e",
+    "address": "4th street"
+  } ],
+  "ingredients": [ {
+    "ingredientName": "Chicken",
+    "quantity": "15",
+    "unit": "whole"
+  } ],
+  "orders": [ {
+    "clientPhone": "94351253",
+    "clientName": "Alice Pauline",
+    "clientAddress": "123, Jurong West Ave 6, #08-111",
+    "recipeName" : "Chicken Rice",
+    "recipeIngredients" : [ {
+      "quantity" : "1",
+      "unit" : "whole",
+      "name" : "Chicken"
+    }, {
+      "quantity" : "200",
+      "unit" : "grams",
+      "name" : "Rice"
+    } ],
+    "price" : "4.50",
+    "deadline" : "12-12-2021 1900",
+    "quantity" : "4",
+    "isFinished" : "no"
+  } ],
+  "recipes": [{
+    "name": "Egg Pr@ta",
+    "price" : "$$2",
+    "recipeIngredients": [{
+      "name": "Egg",
+      "quantity": "2",
+      "unit": "whole"
+    }, {
+      "name": "Flour",
+      "quantity": "100",
+      "unit": "g"
+    }]
+  }]
+}
```

Comment:
Is There should be exactly 3 json files where each json file only has one of the recipe fields invalid.